### DISCUSSION TITLE
Can't use the package from skypack.dev anymore

### DISCUSSION THREAD

**Discussion by Seknarth** [hendt/ebay-api#134] [General]
Hi, I'm using this package in CodePen and importing it via skypack.dev. It was working till yesterday but now it just fails to import it giving me this [error](https://cdn.skypack.dev/error/build:ebay-api@v8.0.0-9XbaNIK7Fj8ldmEEMVEZ). Could you please fix it?

**Comment by dantio** [comment]
Can you downgrade to version 7? I'll check why it's not working with skypack thank you for reporting .

**Reply by Seknarth** [reply]
Ok, I downgraded it and it works correctly now. Waiting for the fix of the v8, thank you :)

**Reply by dantio** [reply]
It looks like it's a problem with skypack, not with this lib. You can raise an issue there.
Actually, the Version 8 was made to emit ESM.
```js
// use the lib own esm for browser
import eBayApi from 'https://cdn.jsdelivr.net/npm/ebay-api@latest/dist/ebay-api.min.mjs';

// this works better than skypack
import eBayEsm from 'https://esm.sh/ebay-api';
```

Do you need the unminified version? I was not aware someone is using the lib in codepen :)

**Reply by Seknarth** [reply]
Thank you for your time, I'll use the esm.sh lib from now on. Best regards!

**Reply by Seknarth** [reply]
Hello again, I'm using esm.sh lib lately but the issue came back today with the v8 even there. Please let me know when you have time if it's only a problem related with both platforms, 'cause in that case I'll raise an issue to skypack and go back to their hosting. Thank you and happy holidays!

**Reply by dantio** [reply]
Thank you for reporting it. I'll revert my last cleanup change and publish in few minutes.

**Reply by dantio** [reply]
It's released now. I hope everything works again for you.

**Reply by Seknarth** [reply]
Ok, it's working again from esm.sh, thank you!
In skypack.dev the issue still persists, just to let you know. Anyway I'll stick to esm.sh for now. Thank you for your time again!
